go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -51,14 +51,6 @@ const (
 )
 
 var (
-	actionableRecoveriesNames = []string{
```

Comment:
`stats.NewCountersWithMultiLabels` does not support a `...string` to init with zero, so this becomes unused